tests/lib/snaps/test-snapd-location-control-provider/provider.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+
+from gi.repository import GLib
+import dbus
+import dbus.service
+
+from dbus.mainloop.glib import DBusGMainLoop
+
+INTERFACE = 'com.ubuntu.location.Service'
+PATH = '/com/ubuntu/location/Service'
+
+DBusGMainLoop(set_as_default=True)
+
+
+class DBusProvider(dbus.service.Object):
+    def __init__(self):
+        bus = dbus.SystemBus()
+        bus_name = dbus.service.BusName(INTERFACE, bus=bus)
+        dbus.service.Object.__init__(self, bus_name, PATH)
+
+    @dbus.service.method(dbus_interface=INTERFACE, out_signature="s")
```

Comment:
The signatures don't match the dbus properties interface for Get and Set: https://dbus.freedesktop.org/doc/dbus-specification.html#standard-interfaces-properties